css-nesting-1/Overview.bs

```diff
@@ -242,12 +313,19 @@ Direct Nesting {#direct}
 		}
 		/* Invalid because & isn't in the first compound selector */
 
+		.foo {
+			color: red;
+			&&.bar { color: blue; }
+		}
+		/* Invalid because the 1st & is no longer 
```

Comment:
Still wrong; the first `&` definitely is at the start of the compound selector. It's the second that's causing the problem. ^_^

Reply:
updated 👍